fontir/src/glyph.rs

```diff
@@ -407,6 +420,19 @@ impl Work<Context, WorkId, WorkError> for GlyphOrderWork {
             }
         }
 
+        // Resolve component references to glyphs that are not retained by conversion to contours
+        // Glyphs have to have consistent components at this point so it's safe to just check the default
+        // See https://github.com/googlefonts/fontc/issues/532
+        for glyph_name in new_glyph_order.clone().iter() {
```

Comment:
hm, I'd normally write this as `iter().cloned()`, which lazily clones each item as needed, instead of cloning the whole collection before iteration? probably slightly better.

Reply:
Amusingly the clone is completely unnecessary; I think I initially anticipated modifying new glyph order in the loop then didn't.